pkg/platform/kube/monitoring/test/function_test.go

```diff
@@ -153,6 +153,20 @@ def handler(context, event):
 			// wait for monitoring
 			time.Sleep(postDeploymentSleepInterval)
 
+			checkInvocationURLs := func(getFunctionOptions *platform.GetFunctionsOptions) {
+				function := suite.GetFunction(getFunctionOptions)
+
+				// function should have external and internal URL enriched
+				suite.Require().Equal(1, len(function.GetStatus().InternalInvocationURLs))
+				suite.Require().Equal(1, len(function.GetStatus().ExternalInvocationURLs))
+
+				suite.Require().Equal("nuclio-function-recover-after-deploy-fail.default.svc.cluster.local:8080",
+					function.GetStatus().InternalInvocationURLs[0])
+				suite.Require().Equal(
+					fmt.Sprintf(":%d", function.GetStatus().HTTPPort),
+					function.GetStatus().ExternalInvocationURLs[0])
+			}
+
```

Comment:
Why is this defined here as a function if it is only called once at the end?

Reply:
@TomerShor added one more usage